src/nxdoc/nuxeo-server/administration/monitoring-and-maintenance/reporting-problems.md

```diff
@@ -169,85 +169,88 @@ history:
 ---
 {{! excerpt}}
 
-Here are listed some procedures to extract information of a running Nuxeo instance. These information can be requested by the support team. Please always compress files before uploading them to your JIRA ticket.
+Here are listed some procedures to extract information of a running Nuxeo instance.
+Most of these scripts generate files in the `/tmp/` directory.
+These information can be requested by the support team.
+Please always compress files before uploading them to your JIRA ticket.
 
 {{! /excerpt}}
 
 &nbsp;
 
-## Nuxeo Configuration and Status
+## {{> anchor 'nuxeo-conf'}}Nuxeo Configuration and Status
 
 To dump your server configuration and status run
 
+```bash
+./bin/nuxeoctl showconf > /tmp/nuxeo-showconf-`date +%Y%m%d-%H%M%S`.txt
+./bin/nuxeoctl status > /tmp/nuxeo-status-`date +%Y%m%d-%H%M%S`.txt
```

Comment:
why is the status needed?

Reply:
you are right the status is part of the showconf